rest/slurmrestapi.py

```diff
@@ -503,5 +515,49 @@ def proxy():
                            url_root=request.url_root,
                            masters=masters)
 
+
+def check_private_data_for_entity(user, entity):
+    """
+    Return true if the entity is one of the attribute defined previously in
+    Private Data settings.
+    """
+    onlyUsersEntities = False
+    currentUser = user
```

Comment:
Please avoid this useless additional variable. The rest of the function code can simply use the user argument directly.